src/TemplateResolutionTrait.php

```diff
@@ -88,6 +87,6 @@ private function getRendererServiceTypeFromContainer(ContainerInterface $contain
             return null;
         }
         $renderer = $container->get(TemplateRendererInterface::class);
-        return get_class($renderer);
+        return $renderer::class;
```

Comment:
Needs dropping PHP 7.4

Reply:
Sorry, I made this change accidentally when running php-cs